src/js/components/ServicesTable.js

```diff
@@ -67,6 +73,30 @@ var ServicesTable = React.createClass({
     this.forceUpdate();
   },
 
+  getServiceLink: function (service) {
+    const id = encodeURIComponent(service.getId());
+
+    if (this.props.isFiltered) {
+      return (
+        <NestedServiceLinks
+          serviceID={id}
+          className="service-breadcrumb"
+          majorLinkClassName="service-breadcrumb-service-id"
+          minorLinkWrapperClassName="service-breadcrumb-crumb" />
+      );
+    }
+
+    return (
+      <Link to="services-detail"
+        className="headline table-cell-value flex-box flex-box-col"
+        params={{id}}>
+          <span className="text-overflow">
```

Comment:
Oh, I've never saw this way of indentation yet in our codebase. 
Good to try new things from time to time, otherwise it gets boring. :P